pkg/ebpf/events_definitions.go

```diff
@@ -6325,4 +6326,17 @@ var EventsDefinitions = map[int32]EventDefinition{
 			{Type: "HookedSyscallData[]", Name: "hooked_syscalls"},
 		},
 	},
+	RegisterChardevRegion: {
+		ID32Bit: sys32undefined,
+		Name:    "register_chrdev_region",
+		Probes: []probe{
+			{event: "__register_chrdev", attach: kprobe, fn: "trace___register_chrdev"},
+		},
+		Sets: []string{},
+		Params: []trace.ArgMeta{
+			{Type: "unsigned int", Name: "major_number"},
+			{Type: "const char*", Name: "char_device_name"},
+			{Type: "u64", Name: "char_device_fops"},
```

Comment:
Pass this argument as a `struct file_operations*` - it would make more sense (and all pointers are treated as `u64` behind the scenes anyway)